cmd/zt_test.go

```diff
@@ -595,13 +595,18 @@ func deleteGCPBucket(c *chk.C, client *gcpUtils.Client, bucketName string, waitQ
 	it := bucket.Objects(ctx, &gcpUtils.Query{Prefix: ""})
 	for {
 		attrs, err := it.Next()
-		if err == iterator.Done {
-			break
+		if err != nil { // if Next returns an error other than iterator.Done, all subsequent calls will return the same error.
+			if err == iterator.Done {
+				break
+			}
+
+			c.Log("Failed to clear GCS bucket:", err) // todo: maybe this code should be more resilient
+			return
```

Comment:
Shouldn't we be returning some sort of failure here. I see function has no return type, feels either we assume it will always succeed or we don't care even if it fails.

Reply:
I'm in agreement there. I was following the paradigm that was already present (it sounded like cleanup failure was intended to be a non-failing error.